### ISSUE TITLE
Adding popular multi-language "restart" phrases?

### ISSUE THREAD

**Issue by mgutt** [JPersson77/LGTVCompanion#132]
For the initial setup the user is forced to select their localized "restart" phrases. Why not collecting some of the popular phrases and built them into the app and totally skip this step to make it even more user-friendly? Finally I think these are "only" 100 to 150 phrases, if all users participate in building a list.

**Comment by JPersson77** [comment]
Hi there, yeah it sounds like a good idea but not really sure how to best go about? Setup a google doc, link to it on the github landing page and ask users to contribute? Whaddyathink?

**Comment by mgutt** [comment]
What about: If the user selects the restart phrase, they will be asked:

```
Do you like to contribute the phrase to enhance the auto-detection in a future release?
[Open website] [Cancel]
``` 

After selecting confirm, a link is opened. This could be a GitHub issue or Google Doc or ...

**Comment by JPersson77** [comment]
Hmm yeah that might work! I'll add it to the todo

**Comment by JPersson77** [comment]
Started some work on this. Please contribute here https://docs.google.com/spreadsheets/d/1SNjYh7kt4aVhHg0ZjI3i96vCeOJZJS73o9Wl-OVEvzk/edit#gid=0

**Comment by JPersson77** [comment]
Implemented in v3.5.0. Feel free to reopen the issue as necessary
